test/e2e/customconfigs/suite_test.go

```diff
@@ -58,7 +58,8 @@ var _ = ginkgo.BeforeSuite(func() {
 	util.WaitForKueueAvailability(ctx, k8sClient)
 	util.WaitForJobSetAvailability(ctx, k8sClient)
 	util.WaitForAppWrapperAvailability(ctx, k8sClient)
-	ginkgo.GinkgoLogr.Info("Kueue, JobSet and AppWrapper operators are available in the cluster", "waitingTime", time.Since(waitForAvailableStart))
+	util.WaitForLeaderWorkerSetAvailability(ctx, k8sClient)
+	ginkgo.GinkgoLogr.Info("Kueue, JobSet, AppWrapper and LeaderWorkerSet operators are available in the cluster", "waitingTime", time.Since(waitForAvailableStart))
```

Comment:
actually, I would prefer not to need to update the log line. iirc we now log also individual times per crd, so the info about installing them is already present, I think here we can say just generic "Required operators are available in the cluster". WDYT?

Reply:
/hold

Reply:
That's a good point. I missed that we're logging individual times per CRD.

Reply:
But maybe we should still log waitingTime to summarize how long it takes overall. WDYT?

Reply:
yeah, total time is also fine